### ISSUE TITLE
[Arcs] Rogue Admiral edge case with Empathy for All

### ISSUE THREAD

**Issue by turmoilbyrd** [LederCards/cards#92]
[Empathy for All](https://cards.ledergames.com/card/ARCS-F1409) is in effect. A player has Rogue Admiral and a Weapon resource. The regent chooses to battle the Empire on a Weapon planet, they choose certain Imperial ships as attacker and some as defender 

There is dispute on how this situation is resolved.

1. If all hits are dealt simultaneously, the outrage triggers only after all the attacker hits are assigned.
2. If hits are dealt sequentially, then the moment they hit one of the defending imperial ship on the Weapon planet, they should trigger Weapon outrage (due to Empathy for All). This makes them lose the Rogue Admiral and now they cannot ignore Truce Law. It would seem they have to abort the rest of the battle?
3. Or maybe they still carry out the rest of the battle since the process has already begun?

There is a related case involving [Navy Reprisal](https://cards.ledergames.com/card/ARCS-F115) hitting imperial ships on a Weapon planet.

**Comment by turmoilbyrd** [comment]
ref discussion https://discord.com/channels/476234833572397056/963112189802201108/1309684879457255538

**Comment by turmoilbyrd** [comment]
Very related #48

**Comment by jdyearsley** [comment]
Yes, Empathy for All should only trigger after the hits are dealt. I feel like this is best addressed by clarifying the baseline idea that all hits from an effect are dealt simultaneously, rather than one at a time?

**Comment by turmoilbyrd** [comment]
> I feel like this is best addressed by clarifying the baseline idea that all hits from an effect are dealt simultaneously, rather than one at a time?

I agree! This also solves another issue, if multiple cities are destroyed in a battle, the city destruction effect for the two cities will only trigger after all the hits area dealt

**Comment by jdyearsley** [comment]
Relevant discussion in #167
